### PR TITLE
Case 20339 - nitpick v1.3.1

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.fogbugz.com/f/cases/20339/
# Description
1.  Online help has been implemented
2.  All entities within a 1000 m radius are deleted by nitpick before a test run.
     This is implemented by running Interface with `deleteNearbyEntities.js` script

# Testing
Tests are to be performed on both Windows and Mac
1.  Run nitpick
2.  Click Help->Online readme
3.  Verify that the nitpick readme.md file has opened in the default browser
4.  Run the full test suite.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16160/HighFidelity-Beta-PR14644-493a05f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16160/HighFidelity-Beta-PR14644-493a05f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16160/HighFidelity-Beta-Interface-PR14644-493a05f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16160/HighFidelity-Beta-PR14644-493a05f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16160/HighFidelity-Beta-Interface-PR14644-493a05f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16200/HighFidelity-Beta-PR14644-dc4be42.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16200/HighFidelity-Beta-Interface-PR14644-dc4be42.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16200/HighFidelity-Beta-PR14644-dc4be42.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16200/HighFidelity-Beta-Interface-PR14644-dc4be42.exe

**Comment by NissimHadar** [comment]
Please build me

**Comment by shanzzam** [comment]
Please build me

**Comment by shanzzam** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16265/HighFidelity-Beta-PR14644-dc4be42.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16265/HighFidelity-Beta-PR14644-dc4be42.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16265/HighFidelity-Beta-Interface-PR14644-dc4be42.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16265/HighFidelity-Beta-PR14644-dc4be42.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16265/HighFidelity-Beta-Interface-PR14644-dc4be42.exe

File: tools/nitpick/src/AWSInterface.cpp [hunk 1]
```diff
@@ -510,31 +519,39 @@ void AWSInterface::updateAWS() {
...
+            stream << "s3.Bucket('hifi-content').put_object(Bucket='" << AWS_BUCKET << "', Key='" << filename << "/" << "Expected Image.png" << "', Body=data)\n\n";
 
-        if (QFile::exists(_htmlSuccessesFolder + "/" + parts[parts.length() - 1] + "/Difference Image.png")) {
-            stream << "data = open('" << _workingDirectory << "/" << filename << "/"
-                   << "Difference Image.png"
-                   << "', 'rb')\n";
+            if (QFile::exists(_htmlSuccessesFolder + "/" + parts[parts.length() - 1] + "/Difference Image.png")) {
+                stream << "data = open('" << _workingDirectory << "/" << filename << "/"
+                    << "Difference Image.png"
+                    << "', 'rb')\n";
```
[the remark this answers is not in the document]
Comment (NissimHadar) [reply tools/nitpick/src/AWSInterface.cpp]: I agree- I will fix all these in the next PR.

**Pull request merged**